build.gradle

```diff
@@ -558,6 +586,7 @@ project(':iceberg-delta-lake') {
     task integrationTest(type: Test) {
       testClassesDirs = sourceSets.integration.output.classesDirs
       classpath = sourceSets.integration.runtimeClasspath
+      jvmArgs += project.property('extraJvmArgs')
```

Comment:
There are a few other places for integration tests we should also add, although they are not run in CI